tests/create_test_data.py

```diff
@@ -0,0 +1,125 @@
+import os
```

Comment:
plz just double check that having a file with the word "test" in the filename inside the tests folder doesn't cause py.test to run this file as part of all the tests when you call `py.test tests/`

Reply:
i renamed it just to be safe